articles/storage/blobs/storage-dotnet-how-to-use-blobs.md

```diff
@@ -20,385 +20,370 @@ ms.author: tamram
 
 [!INCLUDE [storage-selector-blob-include](../../../includes/storage-selector-blob-include.md)]
 
-[!INCLUDE [storage-check-out-samples-dotnet](../../../includes/storage-check-out-samples-dotnet.md)]
+This tutorial will show you how to use [Azure Blob Storage](https://docs.microsoft.com/azure/storage/blobs/storage-blobs-introduction) with ASP.NET and Visual Studio 2017. It will show how to upload, list, download, and delete blobs in the context of a simple Web API project that works with images.
 
-Azure Blob storage is a service that stores unstructured data in the cloud as objects/blobs. Blob storage can store any type of text or binary data, such as a document, media file, or application installer. Blob storage is also referred to as object storage.
+## Get the sample project
 
-### About this tutorial
-This tutorial shows how to write .NET code for some common scenarios using Azure Blob storage. Scenarios covered include uploading, listing, downloading, and deleting blobs.
+To begin, [clone the sample project from GitHub](https://github.com/cartermp/TutorialForStorage). The rest of the tutorial will work with this project.
 
-**Prerequisites:**
+If you're unfamiliar with git, you can also download the project as a zip file. In the GitHub repository, there is a green button called **Clone or download**. You can click that and select **Download ZIP**:
 
-* [Microsoft Visual Studio](https://www.visualstudio.com/)
-* [Azure Storage Client Library for .NET](https://www.nuget.org/packages/WindowsAzure.Storage/)
-* [Azure Configuration Manager for .NET](https://www.nuget.org/packages/Microsoft.WindowsAzure.ConfigurationManager/)
-* An [Azure storage account](../common/storage-create-storage-account.md?toc=%2fazure%2fstorage%2fblobs%2ftoc.json#create-a-storage-account)
+![Screenshot of GitHub showing how to clone or download the sample application](media/storage-blobs-introduction/download-sample.png)
 
-[!INCLUDE [storage-dotnet-client-library-version-include](../../../includes/storage-dotnet-client-library-version-include.md)]
+## Set up your environment
 
-### More samples
-For additional examples using Blob storage, see [Getting Started with Azure Blob Storage in .NET](https://azure.microsoft.com/documentation/samples/storage-blob-dotnet-getting-started/). You can download the sample application and run it, or browse the code on GitHub.
+Ensure that you have [Visual Studio 2017](https://www.visualstudio.com/vs/) installed with the **ASP.NET and web development** and **Azure Development** workloads. This will give you all of the tools that you will need for this tutorial.
```

Comment:
Is there a way to make this doc work for multiple IDE setups? Such as VS for Mac and VS Code?

Reply:
Perhaps, but I feel like that will complicate the doc. We'll have to sync with the docs team on what their strategy is for this sort of stuff, especially since tools change so often. They may want to not even make mention of any tool and be strictly documenting their APIs.